rust-src/ecvrf/src/secret.rs

```diff
@@ -1,19 +1,14 @@
 //! ed25519 secret key types.
 
+use crate::{constants::*, errors::*, public::*};
+use anyhow::Result;
 use core::fmt::Debug;
-
-use zeroize::Zeroize;
-
+use crypto_common::*;
 use curve25519_dalek::{constants, digest::Digest, scalar::Scalar};
-
 use rand::{CryptoRng, Rng};
-
 use sha2::Sha512;
-
 use subtle::{Choice, ConstantTimeEq};
-
-use crate::{constants::*, errors::*, public::*};
-use crypto_common::*;
+use zeroize::Zeroize;
```

Comment:
Remove the empty lines preceding this one. Collapse all imports without empty lines and then fmt.

Reply:
That's what I did already?